moler/cmd/unix/iperf2.py

```diff
@@ -64,12 +64,13 @@ class Iperf2(GenericUnixCommand, Publisher):
       ("192.168.0.10", "5016@192.168.0.12"): {'report': {<report dict here>}}
 
     """
-    def __init__(self, connection, options, prompt=None, newline_chars=None, runner=None):
+    def __init__(self, connection, options, path='', prompt=None, newline_chars=None, runner=None):
```

Comment:
Already done in base class https://github.com/nokia/moler/blob/master/moler/cmd/commandtextualgeneric.py 
` self.command_path = None  # path to command executable - allow non standard locations like /usr/local/bin/`

Reply:
Thank you! Pull request to close.